jar/src/cljc/sixsq/slipstream/client/api/utils/http_utils.cljc

```diff
@@ -0,0 +1,16 @@
+(ns sixsq.slipstream.client.api.utils.http-utils)
+
+(def http-lib-insecure-key :kvlt.platform/insecure?)
+
+(defn- process-insecure
+  [req]
+  (if (contains? req :insecure?)
+    (-> req
+      (merge {http-lib-insecure-key (:insecure? req)})
+      (dissoc :insecure?))
+    req))
+
+(defn process-req
```

Comment:
Here I just thought about chaining request "processors" in the future :)  If you have a strong opinion about merging process-insecure into process-req - then I'll do that.